configs/components/augeas.rb

```diff
@@ -37,7 +39,11 @@
       pkg.build_requires 'runtime'
     end
 
-    pkg.build_requires 'pkgconfig'
+    if platform.architecture == "s390x"
+      pkg.environment "PATH" => "/opt/pl-build-tools/bin:$$PATH:#{settings[:bindir]}"
+      pkg.environment "CFLAGS" => settings[:cflags]
+      pkg.environment "LDFLAGS" => settings[:ldflags]
```

Comment:
Totally agree, this is something to discuss next week when we have a work session to talk about refactoring the p-a config files, where I'll be dumping some ideas out of my head that have been rattling around for a while.